dashboard/src/features/accessControl/components/EditPolicies.jsx

```diff
@@ -15,10 +15,14 @@ class EditPolicies extends React.Component {
     return(
       <div className={styles.main}>
         {policyOptions.map(option => {
+          const grant = this.props.item.grants.find(g => g.policy == option.value)
```

Comment:
I believe the backend implementation treats `{policy: "foo", protected: false}` and `{policy: "foo", protected: true}` as separate grants. While this is an unlikely edge case, I think it's a good idea to try to account for it here.

One possibility is to have the edit checkboxes correspond to the unprotected versions of the grants, and to list any protected grants that may appear under the guard as non-interactive "footnotes" in the editing interface.